src/ui/components/Transcript/Transcript.tsx

```diff
@@ -38,7 +46,7 @@ function Transcript({ recordingId, pendingComment }: PropsFromRedux) {
           </div>
         ) : (
           <div className="transcript-list space-y-4 text-lg text-gray-500">
-            None yet! Please click on the video to place the first comment.
+            {emptyStateText()}
```

Comment:
You could just do this inline so that it's cleaner like so:

```js
<div className="transcript-list space-y-4 text-lg text-gray-500">
  {isAuthenticated ? 'None yet! Please click the video to add a comment.' : 'Please log in to add a comment to this replay.'}
</div>
```

Reply:
Thanks! I had some trouble making that work, but glad to know it's possible. Will modify.